packages/grafana-ui/src/slate-plugins/suggestions.tsx

```diff
@@ -289,17 +290,15 @@ const handleTypeahead = async (
       if (!group.items) {
         return group;
       }
-
+      const searchFunctionType =
```

Comment:
I would add a comment on what does the code after `||` mean and why do we have it (backward compatibility)